opam/win32/compilers/4.02.3+mingw64.comp

```diff
@@ -0,0 +1,26 @@
+opam-version: "1"
+version: "4.02.3"
+src: "http://caml.inria.fr/pub/distrib/ocaml-4.02/ocaml-4.02.3.tar.xz"
+patches:[
+"https://github.com/djs55/ocaml/commit/e5ff15e7b028445bcd5da88f90874bcc1e4597ac.diff"
```

Comment:
(the patch in that list are applied in sequence before the compiling is built ?)